src/iden3comm/packers/zkp.ts

```diff
@@ -174,6 +175,49 @@ export class ZKPPacker implements IPacker {
   mediaType(): MediaType {
     return MediaType.ZKPMessage;
   }
+
+  /** {@inheritDoc IPacker.getEnvelope} */
+  getEnvelope(): string {
```

Comment:
and in future I guess we will have potentially an array of strings even for one media type, maybe we should return an array of strings and rename methods to getAcceptedProfiles()